src/musicbrainz/chromaprinter.cpp

```diff
@@ -143,16 +143,24 @@ QString Chromaprinter::CreateFingerprint() {
   ChromaprintContext* chromaprint =
       chromaprint_new(CHROMAPRINT_ALGORITHM_DEFAULT);
   chromaprint_start(chromaprint, kDecodeRate, kDecodeChannels);
-  chromaprint_feed(chromaprint, reinterpret_cast<void*>(data.data()),
+  chromaprint_feed(chromaprint, reinterpret_cast<int16_t *>(data.data()),
```

Comment:
This compiles well on my GCC. There is an implicit conversion `int16_t* -> void*`.

See http://en.cppreference.com/w/cpp/language/implicit_conversion (**Pointer conversions**).